presto-main/src/main/java/com/facebook/presto/operator/TaskContext.java

```diff
@@ -757,4 +768,16 @@ private <T extends PlanNode> Optional<T> findPlanNode(PlanNodeId planNodeId, Cla
                 .where(node -> node.getId().equals(planNodeId) && nodeType.isInstance(node))
                 .findSingle();
     }
+
+    public LongSet getCompletedSplitSequenceIds()
```

Comment:
Are the following three files mainly to support this use of LongSet ? Does this need to be a set ? Can it be an int[] array that is deduped here during creation ?